packages/plugins/table/src/index.tsx

```diff
@@ -2,26 +2,27 @@ import { StatefulPlugin, string } from '@edtr-io/plugin'
 import * as React from 'react'
 import ReactMarkdown from 'react-markdown'
 
-import { createTableEditor } from './editor'
+import { TableEditor } from './editor'
 
 export const tableState = string()
 
 export function createTablePlugin(
-  config: TablePluginConfig
-): StatefulPlugin<typeof tableState> {
+  renderMarkdown = defaultRenderMarkdown
+): StatefulPlugin<typeof tableState, TablePluginConfig> {
```

Comment:
(Possibly) different from the options passed to the factory, the plugin defines its config (convention: named `FooPluginConfig`